course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -215,7 +215,7 @@
                 }
             });
 
-            modalInstance.result.then(function modalConfirmed() {
+            $scope.modalInstance.result.then(function modalConfirmed() {
```

Comment:
Yes, I had it in my earlier version and put it while applying my stashed changes.